bigtable/admin/CMakeLists.txt

```diff
@@ -48,3 +51,25 @@ foreach (fname ${bigtable_admin_unit_tests})
     add_test(NAME ${target} COMMAND ${target})
     add_dependencies(tests-local ${target})
 endforeach ()
+
+install(TARGETS bigtable_admin_client
+    EXPORT bigtable-targets
+    RUNTIME DESTINATION bin
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(DIRECTORY admin/ DESTINATION include/bigtable/admin FILES_MATCHING PATTERN "*.h")
```

Comment:
Should this be /include/google/bigtable?

Reply:
**tl;dr;** I prefer the current directory hierarchy, but if there is a reason why we should use `google/bigtable` now is the time to change all of it.

The code, tests, and examples use

```C++
#include "bigtable/admin/foo.h"
```

I believe we want the users to use the same include paths.  So if we decide they should be installed in `include/google/bigtable`, then the users would need to say:

```C++
#include <google/bigtable/admin/foo.h> // or with quotes, up to them
```

But then we would need to move all the files in `google-cloud-cpp/bigtable` to `google-cloud-cpp/google/bigtable` or `google-cloud-cpp/bigtable/google/bigtable`.  I guess we could ask the users to add `-I /usr/local/include/google` to their paths, but that would make the includes for protobuf ambiguous (you could include them with `google/` or without)

Reply:
Will defer to your preference for the current hierarchy. I wanted to ensure that missing google was not an oversight.